hpx/util/unwrapped.hpp

```diff
@@ -127,12 +132,11 @@ namespace util {
     ///             as soon as we allowed to route non future types through.
     ///
     template <typename... Args>
-    // This will be enabled once the old implementation is deprecated:
-    // #if !defined(HPX_NO_DEPRECATE_UNWRAPPED)
-    //     HPX_DEPRECATED("hpx::util::unwrapped was replaced by "
-    //                    "hpx::util::unwrap and hpx::util::unwrapping "
-    //                    "and might be removed in a later version of HPX!")
-    // #endif    // HPX_NO_DEPRECATE_UNWRAPPED
+#if !defined(HPX_NO_DEPRECATE_UNWRAPPED)
```

Comment:
@hkaiser This preprocessor constant is required for the `unwrapped.cpp` unit test, which still uses the old implementation for testing. Since I didn't want to yield deprecation messages from the said unit test, I introduced the conditional deprecation message.
As alternative I could remove the old unit test.